contributors/guide/README.md

```diff
@@ -23,15 +23,15 @@ Welcome to Kubernetes!
     -   [Find something to work on](#find-something-to-work-on)
         -   [Find a good first topic](#find-a-good-first-topic)
         -   [Learn about SIGs](#learn-about-sigs)
+        -   [SIG-specific contributing guidelines](#sig-specific-contributing-guidelines)
```

Comment:
i think the TOC was not generated properly. the Learn about SIGs topic should have 3 children
1. sig structure
2. find a sig...
2.1 SIG-specific contribution guides

something else must be messed up because file an issue has a different heading too

Reply:
Hm. We hadn't previously been using any headers smaller than those anyway. :/ I'll actually make it a taller heading so the ToC makes sense.

I think `File an Issue` is the correct level though?